stubs/requests/requests/adapters.pyi

```diff
@@ -2,9 +2,11 @@ from collections.abc import Mapping
 from typing import Any
 
 from urllib3 import exceptions as urllib3_exceptions, poolmanager, response
+from urllib3.contrib import socks
 from urllib3.util import retry
 
 from . import cookies, exceptions, models, structures, utils
+from .compat import basestring as basestring
 
 Response = models.Response
```

Comment:
For lines 11-34: All of these are re-exports at runtime. I'm not quite sure why we're using assignments in the stub -- re-exports are generally less buggy, so I think we should use re-exports here as well, like  the runtime does.